docs/technical_reference/unit_models/crystallizer_0D.rst

```diff
@@ -0,0 +1,121 @@
+
+Crystallization (0D)
+=======================
+
+Introduction
+-------------
+Crystallization from solution is a separation technique where a solid phase is separated from a mother
+liquor (Kramer et al. [1]).
+
+The model implemented here is for a continuous evaporative crystallization. The main features of the implemented model are as follows:
+
+1) Evaporative crystallization is modeled as shown in Figure 1.
+2) Model dimensionality is limited to 0D.
+3) Model dynamics are restricted to steady-state only.
+4) The unit supports solid-liquid-vapour mixtures.
+5) The model has only been demonstrated for single solutes only.
+6) The temperature in the crystallizer is fixed.
+7) The exit liquid is assumed to lose its supersaturation.
+
+.. figure:: ../../_static/unit_models/crystallizer.jpg
+    :width: 400
+    :align: center
+    
+    Figure 1. Schematic representation of the evaporative crystallization configuration system modeled in IDAES, comprising of a crystallization unit (C), a heater (Q) and a separator (S). The broken green box shows the system boundary. 
+
+Ports
+---------
+
+The model provides four ports (Pyomo notation in parenthesis):
+
+* Inlet port (inlet)
+* Liquid outlet port (outlet)
+* Solid crystals port (solids)
+* Vapor outlet port (vapor)
+
+
+Degrees of Freedom
+-------------------
+Aside from the inlet feed state variables (i.e. temperature, pressure, component flowrates), the crystallizer model has
+at least 5 degrees of freedom that should be fixed for the unit to be fully specified.
+
+Typically, the following four variables are fixed, in addition to state variables at the inlet (Pyomo notation in parenthesis):
+
+    * Crystal growth rate, :math:`G` (crystal_growth_rate)
+    * Desired median length of solid crystals, :math:`L_D` (crystal_median_length)
+    * Parameter for Sounders-Brown relation, :math:`C_v` (souders_brown_constant)
+    * Crystallization temperature (temperature_operating)
+
+In addition to those four variables, the user may specify any of the following variables to fully specify the unit (Pyomo notation in parenthesis):
+
+    * Crystallization yield (crystallization_yield['NaCl'])
+    * Solids volumetric fraction of product slurry, :math:`1-{\varepsilon}` (product_volumetric_solids_fraction)
+    * Magma density, :math:`{M}_{T}` (dens_mass_magma)
+    * Solid outlet mass flow
+    * Vapour outlet mass flow
+
+
+Variables
+----------
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
+
+   "Crystal growth rate", ":math:`G`", "crystal_growth_rate", "None", ":math:`\text{m/s}`"
+   "Sounders-Brown constant", ":math:`C_v`", "souders_brown_constant", "None", ":math:`\text{dimensionless}`"
```

Comment:
```suggestion
   "Sounders-Brown constant", ":math:`C_v`", "sounders_brown_constant", "None", ":math:`\text{dimensionless}`"
```
Is the variable name spelled with or without the "n" in the unit model file?

Reply:
Corrected.